### ISSUE TITLE
[Weblate] Unable to start new translation

### ISSUE THREAD

**Issue by Generator** [TBog/TBLauncher#358]
# Description #

Trying to translate the app to pt-PT, but the language isn't available for 'strings' and 'strings-26'.
When trying to create a new language reverts to login page.

## Context ##

This is a known issue with Weblate, only admins can create languages

**Comment by TBog** [comment]
I'm not sure how to setup the translation for a strings file to be used only on API26.

My current though on this is to remove the strings-v26 file and add code to switch the text.

**Comment by Generator** [comment]
In any case im only able to translate 'metadata'. Can't access strings.

Having two types of strings is just extra work and inconsistent translations. Better ditch 'strings-26' and find another way to use it.

**Comment by Generator** [comment]
No, is the language isn't available, and users can create a new language. 
There's only 'pt' (Portuguese) and not pt-PT (Portuguese (Portugal)), but metadata have pt-PT.

The problem with using 'pt' for translation is that includes Brazilian (wich already exists) and other portuguese languages. If someone tries to translate a word to Portuguese (Portugal) and then other one translates same word to Brazilian Portuguese, translations become inconsistent and a nesses.

That's an issue with Weblate, not user settings. Already had found somewhere about it when had the same issue with other project, only admins can add new languages in some cases.
